spec/helpers/applications_helper_spec.rb

```diff
@@ -85,6 +85,12 @@
     end
   end
 
+  describe "decode" do
+    it "should decode the html entities such as &amp; to &" do
```

Comment:
For some reason my instincts are telling me we need more tests around the edge cases. My instincts could be wrong :smile: 

Are there situations where we wouldn't want to decode HTML entities? Are the potential security issues we could be introducing to the page?

It might be good to have a look on PlanningAlerts for real-world examples of where this is a problem and specifically test them too. I always think if you've found a problem it's best to add tests for that exact problem, and the other success, failure, and edge cases you can think of.